File: FizzBuzz/fizzbuzz.R
#!/usr/bin/env Rscript

for (i in seq(from=1, to=100,by=1)) {
  if (i %% 15 == 0){ 
    print("FizzBuzz")
  } else if (i %% 3 == 0) {
    print("Fizz")
  } else if (i %% 5 == 0) {
    print("Buzz")
  } else {
    print(i)
  }
}

File: FizzBuzz/fizzbuzz.rb
#!/usr/bin/env ruby

for counter in 1..100
  if counter % 15 == 0 then
    puts "fizzbuzz"
  elsif counter % 5 == 0 then
    puts "buzz"
  elsif counter % 3 == 0 then
    puts "fizz"
  else
    puts counter
  end
end
